packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/EnterAmount/template.success.tsx

```diff
@@ -42,23 +52,62 @@ const MinMaxContainer = styled.div`
   margin: 24px 0 40px;
   min-height: 30px;
   display: flex;
+  justify-content: center;
 `
 const CoinContainer = styled.div`
   display: flex;
   flex-direction: row;
 `
+
+const PendingText = styled(Text)`
+  a {
+    color: ${props => props.theme.blue600};
+    text-decoration: none;
+  }
+`
 const CustomBlueCartridge = styled(BlueCartridge)`
   cursor: pointer;
 `
 const CustomErrorCartridge = styled(ErrorCartridge)`
   cursor: pointer;
 `
-const PendingText = styled(Text)`
-  a {
-    color: ${props => props.theme.blue600};
-    text-decoration: none;
-  }
+
+const Limits = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 15px 40px;
+  border-top: 1px solid ${props => props.theme.grey000};
+  border-bottom: 1px solid ${props => props.theme.grey000};
+`
+
+const LimitWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+`
+const FiatIconWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+`
+const AmountRow = styled(Row)`
+  position: relative;
+  padding: 24px;
+  justify-content: center;
+  border: 0;
+`
+const SubIconWrapper = styled.div`
+  align-items: center;
+  justify-content: center;
```

Comment:
if this doesn't have `displaly: flex` `justify-content & align-items` won't do anything here.